Source/WebCore/inspector/InspectorOverlay.cpp

```diff
@@ -1441,7 +1441,7 @@ static Vector<String> authoredGridTrackSizes(Node* node, GridTrackSizingDirectio
         }
     }
 
-    auto* cssValueList = dynamicDowncast<CSSValueList>(*cssValue);
+    auto* cssValueList = dynamicDowncast<CSSValueList>(cssValue.get());
```

Comment:
do we need to `.get()`?  or can we just `dynamicDowncast<CSSValueList>(cssValue)`?

Reply:
Yeah, we need the `.get()` here, otherwise we try to downcast the RefPtr itself to CSSValueList, which doesn't compile 😅